numpy/add_newdocs.py

```diff
@@ -5919,6 +5919,13 @@ def luf(lamdaexpr, *args, **kwargs):
         First array
     B : array_like
         Second array
+    dtype : data-type code, optional
+        The type used to represent the intermediate results. Defaults
```

Comment:
Totally missed the mark with this one - the arguments are just forwarded to the ufunc, including things like `where` which don't have all too sensible a result